datahub-upgrade/src/main/java/com/linkedin/datahub/upgrade/system/cdc/debezium/ConfigureDebeziumConnectorStep.java

```diff
@@ -0,0 +1,334 @@
+package com.linkedin.datahub.upgrade.system.cdc.debezium;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.linkedin.datahub.upgrade.UpgradeContext;
+import com.linkedin.datahub.upgrade.UpgradeStep;
+import com.linkedin.datahub.upgrade.UpgradeStepResult;
+import com.linkedin.datahub.upgrade.impl.DefaultUpgradeStepResult;
+import com.linkedin.metadata.config.DebeziumConfiguration;
+import com.linkedin.metadata.config.EbeanConfiguration;
+import com.linkedin.metadata.config.kafka.KafkaConfiguration;
+import com.linkedin.upgrade.DataHubUpgradeState;
+import io.datahubproject.metadata.context.OperationContext;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.autoconfigure.kafka.KafkaProperties;
+
+/**
+ * Step that configures and creates the Debezium connector for CDC processing. Builds the connector
+ * configuration from available sources and posts it to Kafka Connect.
+ */
+@Slf4j
+public class ConfigureDebeziumConnectorStep implements UpgradeStep {
```

Comment:
Same comment as above, objectmapper, remove python conventions. Apply to all classes.